.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,21 +1,27 @@
-### Purpose of PR:
-Please give a short description of what this PR is for.
+### Purpose of PR  
+<!-- Describe what this PR does. -->
 
+### Linked Issues  
+<!-- Add links to related issues or JIRA tickets. -->
+- Fixes MAHOUT-XXXX  
+- Related to #123  
 
-### Important ToDos
-Please mark each with an "x"
-- [ ] A JIRA ticket exists (if not, please create this first)[https://issues.apache.org/jira/browse/mahout/]
-- [ ] Title of PR is "MAHOUT-XXXX Brief Description of Changes" where XXXX is the JIRA number.
-- [ ] Created unit tests where appropriate
-- [ ] Added licenses correct on newly added files
-- [ ] Assigned JIRA to self
-- [ ] Added documentation in scala docs/java docs, and to website
-- [ ] Successfully built and ran all unit tests, verified that all tests pass locally.
+### Changes Made  
+- [ ] Bug fix  
+- [ ] New feature  
+- [ ] Documentation update  
 
-If all of these things aren't complete, but you still feel it is
```

Comment:
Add something here about making it a draft pr (github's updated version of what this is describing)